test/Reporter.test.js

```diff
@@ -56,8 +57,9 @@ describe("Reporter", function() {
 
 			plugins.done(simpleStats);
 			setTimeout(function() {
+
 				should.strictEqual(console.log.callCount, 2);
-				should.strictEqual(console.log.calledWith("webpack: Compiled successfully."), true);
+				should.strictEqual(console.log.calledWith(getTime() + "webpack: Compiled successfully."), true);
```

Comment:
these tests shouldn't be modified, they should be left as-is and new tests created when the option is set.

also, there shouldn't be a need for a separate `GetTime` module. you can use a regular expression to check that the timestamp is being prepended to the message.